src/LoopChecker.java

```diff
@@ -0,0 +1,34 @@
+public class LoopChecker implements Checker {
+
+    @Override
+    public void check(String login, String password, String confirmPassword) throws WrongPasswordException, WrongLoginException {
+        checkLength(login, true);
+        checkLength(password, false);
+        checkCharacters(login, true);
+        checkCharacters(password, false);
+        checkPassword(password, confirmPassword);
+    }
+
+    private void checkCharacters(String s, boolean login) throws WrongLoginException, WrongPasswordException {
+        if (s == null) {
+            throwException(login, "null");
+        }
+        char[] chars = s.toCharArray();
+        for (char Ch : chars) {
```

Comment:
```suggestion
        for (char ch : chars) {
```
названия переменных надо писать с маленькой буквы